### ISSUE TITLE
[question] : Why is the liferay-maven plugin present in the pom.xml  ?

### ISSUE THREAD

**Issue by Niceplace** [vernaillen/liferay7-springmvc-portlet#2]
Hi,

From my (limited) understanding of Liferay 7,[the liferay-maven plugin will not be maintained anymore.](https://web.liferay.com/web/david.truong/blog/-/blogs/the-state-of-maven-development-in-liferay)
Also, with the introduction of the OSGI standard, wouldn't portlets need to be deployed only into Apache Felix, and not in the container (Tomcat, or something else) ? 

Is it ok to remove the liferay-maven plugin from the pom.xml ?

**Comment by vernaillen** [comment]
You are absolutely right. I was still using this plugin to deploy the war to the liferay deploy, but that's just a file copy. 
So, this plugin is indeed not needed.

I still need to review and test the new Liferay 7 maven plugins and then update this project, so i will leave this issue open for now.
